contrib/jenkins_tests/style.sh

```diff
@@ -66,9 +76,7 @@ else
 fi
 rc=$(($rc+$nerrors))
 
-module unload "dev/clang-9.0.1"
-
 do_archive "${style_dir}/*.diff"
 
 echo "[${0##*/}]..................exit code = $rc"
-exit $rc
+exit $rc
```

Comment:
Slava, this is getting ridicules, every PR we give you the same comment of this missing new line, please configure your VS-Code properly to NOT remove newlines on save.

Reply:
Fixed